dotcom-rendering/src/web/browser/ga/index.ts

```diff
@@ -22,7 +22,16 @@ export const ga = (): Promise<void> => {
 					init();
 					sendPageView();
 				})
-				.catch((e) => console.error(`GA - error: ${String(e)}`));
```

Comment:
Interesting, so does Sentry record whenever `console.error` is called?

Reply:
I believe so, yes